languages/elixir/config.json

```diff
@@ -160,6 +160,12 @@
           "default-arguments"
         ]
       },
+      {
+        "slug": "exceptions",
+        "uuid": "9f698b46-fe8d-4641-a928-bcebfd9678df",
+        "concepts": "exceptions",
+        "prerequisites": ["access-behaviour", "errors"]
```

Comment:
Was there any organization-wide agreement about how specific we need to be listing prerequisites?

Technically, this one should also require strings, structs, lists, multiple clause functions, guards. I definitely see how it's redundant to list all of that for every exercise further in the track... But I am also worried that if we always make assumptions that certain prerequisites can be omitted because they are also indirectly prerequisites of other prerequisites, then the track might not be very easy to reshape by exchanging a concept exercise for a simpler one (with fewer prerequisites).

Reply:
I think slack has consumed the reference. I had asked ihid this, and I think we had decided that we can list all possible concepts, but we only _have to_ list the closest ancestor.

Is that correct, @iHiD